README.md

```diff
@@ -16,6 +16,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [Critical TechWorks](https://www.criticaltechworks.com/) [:rocket:](https://www.criticaltechworks.com/#openroll) | Developing next generation software systems for BMW. | `Lisboa` `Porto` |
 | [Mercedes-Benz.io](https://www.mercedes-benz.io) [:rocket:](https://www.mercedes-benz.io/jobs) | Shaping the digital future of Mercedes-Benz. | `Lisboa` |
 | [Stratio Automotive](https://stratioautomotive.com) [:rocket:](https://stratio.workable.com) | Vehicle data into predictive intelligence. | `Coimbra` |
+| [Daimler Trucks & Buses Tech Data Hub](https://techhublisbon.io/) [:rocket:](https://techhublisbon.io/#jobs) | Trucks and Buses IoT data into intelligent global services. | `Lisboa` |
```

Comment:
Hi @nunomrc, is it possible to shorten the company name? Or else it will break the table layout.  
E.g. `DTB Tech & Data Hub` (copied from the website title) or something alike.

Also, can you please make sure companies are sorted alphabetically?

Thanks!

Reply:
Added a new commit with these requests. Thanks!